dnf/base.py

```diff
@@ -53,9 +53,9 @@
 import dnf.logging
 try:
     import dnf.module.module_base
-    WITH_MODULES = True
+    WITH_MODULES = True  # :api
```

Comment:
Please for what reason they use it?

Reply:
To detect availability of modularity. Changed the comment, I will propose ansible to use the same way as in dnf: try to import dnf.module